models/bgrid_solo/readme.rst

```diff
@@ -58,6 +89,28 @@ Program ``ps_rand_local`` generates a set of randomly located surface pressure
 observations with an interactively specified error variance. It also allows the
 observations to be confined to a rectangular subdomain.
 
+Diagnostics
+-----------
+
+The best method to determine the performance of an experiment in which you
+assimilate data from real-world sources is to compare the ensemble estimates of
+the observation to your real-world data. You can estimate the bias and error of
+the ensemble mean or gauge how many of the real-world observations are actually
+being assimilated. These diagnostics are known as observation-space diagnostics.
+DART provides the program ``obs_diag`` and MATLAB observation space diagnostics
+for you to use to quickly assess the performance of your experiment. 
+
+``obs_diag`` reads the ``obs_seq.final`` files and calculates several quantities,
+such as the root-mean-squared error, bias, and spread, for an arbitrary number of
+regions and levels. ``obs_diag`` outputs a netCDF file called ``obs_diag_output.nc``,
+which can then be used in the DART MATLAB routines located in
+``DART/diagnostics/matlab/``.
+
+For more detail on obs_diag and the MATLAB diagnostics, see `PROGRAM obs_diag
+(threed_sphere) <https://docs.dart.ucar.edu/en/latest/assimilation_code/programs
+/obs_diag/threed_sphere/obs_diag.html>`_ and `MATLAB observation space diagnostics
+<https://docs.dart.ucar.edu/en/latest/guide/matlab-observation-space.html>`_
+
```

Comment:
I know Kevin is reviewing this, but this is a definite change. Don't hardcode the html page as the link:

https://docs.dart.ucar.edu/en/latest/assimilation_code/programs/obs_diag/threed_sphere/obs_diag.html

Use the section headings or the custom anchors

Reply:
Can we use a section heading on a different page?
Is that how custom anchors would be used?

Would it work (long term) to use relative path names?
E.g. ../../assimilation_code/programs/obs_diag/threed_sphere/obs_diag.html

Reply:
so yes and no for the relative paths
you don't want ../../assimilation_code/programs/obs_diag/threed_sphere/obs_diag.html
with .html because you might not be building html. 
you can use :doc:../../assimilation_code/programs/obs_diag/threed_sphere/obs_diag (no extension) 
with the relative path. 

Your link will break if the .rst file is moved. vs. if you link to the anchor the link will still work if you rearrange the files. 

also you want to be really careful to not hardcode the version, e.g. 
https://docs.dart.ucar.edu/en/**latest**/assimilation_code/programs
/obs_diag/threed_sphere/obs_diag.html
because then the link takes you the hardcoded version, rather than the page within the version of the documentation you are looking at. 
e.g for this pull request the documentation is here https://dart-documentation--689.org.readthedocs.build/en/689/
but the link takes you to https://docs.dart.ucar.edu/en/latest/

I know we have all sorts of links of various flavours all over the documentation, but I think it is worth being careful in pull requests to not introduce